app/src/main/java/org/mozilla/fenix/home/recenttabs/view/RecentTabs.kt

```diff
@@ -67,7 +76,8 @@ import org.mozilla.fenix.theme.FirefoxTheme
 fun RecentTabs(
     recentTabs: List<RecentTab>,
     onRecentTabClick: (String) -> Unit = {},
-    onRecentSearchGroupClicked: (String) -> Unit = {}
+    onRecentSearchGroupClicked: (String) -> Unit = {},
+    menuItems: List<RecentTabMenuItem>
```

Comment:
Can we move menuItems below recentTabs? I prefer to keep the callback functions closer to bottom of the parameter list with the exception of `@Composeable content`

Reply:
Yep! Missed it on my readthrough, thanks. Updated throughout